neqo-qpack/src/huffman.rs

```diff
@@ -116,7 +116,7 @@ fn decode_character(reader: &mut BitReader) -> Res<Option<u16>> {
 /// Never, but rust doesn't know that.
 #[must_use]
 pub fn encode(input: &[u8]) -> Vec<u8> {
-    let mut output: Vec<u8> = Vec::new();
+    let mut output: Vec<u8> = Vec::with_capacity(input.len()); // Huffman compresses, so this should be big enough to not reallocate.
```

Comment:
It can reallocate when given pathological data, but we don't do that.  So this is probably fine.